Documentation/edgefs-cluster-crd.md

```diff
@@ -164,6 +164,12 @@ The following storage selection settings are specific to EdgeFS and do not apply
   - `3`: Most durable and reliable option at the cost of significant performance impact.
 - `maxSizeGB`: For `rtlfs`, defines maximum allowed size to use per directory in gigabytes. For `rtkvs` this is the maximum space the disk's metadata table can occupy.
 - `zone`: Enables the node's failure domain number. Default value is 0 (no zoning). Zoning number is a logical failure domain tagging mechanism and if enabled then it has to be set for all the nodes in the cluster. See also, the `failureDomain`
+- `payloadS3URL`: When set, it activates payload data chunk forwarding to an external S3 server. The value is an URL of the S3 bucket the payload chunks will be put to. Example: http://s3.aws-region.amazonaws.com/bucket. Only applicable for RTRD (raw disk) engine. Disabled by default.
+- `payloadS3Region`: S3 server region. Only applicable when the `payloadS3URL` is set.
+- `payloadS3MinKb`: a minimal payload chunk size to trigger the forwarding to the S3 bucket. Data chunks smaller than this value will be stored locally. Only applicable when the `payloadS3URL` is set.
+- `payloadS3Capacity`: capacity of the external S3 bucket expressed in bytes. Only applicable when the `payloadS3URL` is set.
```

Comment:
I understand this is "artificial" capacity so that EdgeFS can report accurate usage summary. I suggest we mention that this is artificial and also allow it to be set in GB.

Reply:
Updated accordingly